onsite/phosphors/cli.py

```
@@ -253,11 +253,27 @@ def phosphors(
+            if orig_pf:
+                orig_pf_map[pep_idx] = orig_pf
+        for row in result_rows:
+            pep_idx = row.get("peptide_identification_index")
+            orig_pf = orig_pf_map.get(pep_idx, "")
+            out_pf = str(row.get("peptidoform", ""))
+            if orig_pf and orig_pf != out_pf:
+                relocated_count += 1
+
         # Report
         elapsed = time.time() - start_time
         click.echo(f"\nProcessing Complete:")
```

Ruff (0.15.17)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)